gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-scim/src/main/java/io/gravitee/am/gateway/handler/scim/mapper/UserMapper.java

```diff
@@ -119,6 +121,24 @@ public static User convert(io.gravitee.am.model.User user, String baseUrl, boole
         meta.setLocation(baseUrl + (listing ? "/" + scimUser.getId() : ""));
         scimUser.setMeta(meta);
 
+        // Enterprise User
+        if (user.isEnterpriseUser()) {
+            EnterpriseUser.EnterpriseUser0 enterpriseUser = new EnterpriseUser.EnterpriseUser0();
+            enterpriseUser.setEmployeeNumber(user.getEmployeeNumber());
+            enterpriseUser.setCostCenter(user.getCostCenter());
+            enterpriseUser.setOrganization(user.getOrganization());
+            enterpriseUser.setDivision(user.getDivision());
+            enterpriseUser.setDepartment(user.getDepartment());
+            if (user.getManager() != null) {
+                Manager manager = new Manager();
+                manager.setValue(user.getManager().getValue());
+                manager.setRef(user.getManager().getRef());
+                manager.setDisplayName(user.getManager().getDisplayName());
+                enterpriseUser.setManager(manager);
+            }
```

Comment:
question: should we check that the manager exists in our DB or in the current Bulk user list ? 🤔

Reply:
🤔 , don't know if it's "too much", or should we just throw a 404 if the user try to find its manager when doing a get ?